### PR TITLE
[i18n/audio] Add more CDF election string extractors

### PR DESCRIPTION
**Pull request by kofi-q**
## Overview

_Related spec [here](https://docs.google.com/document/d/1a2GRj1g4aOawEssPdTCgbv-qs28d-M_dyMEfzm4I4nM/edit?usp=sharing)._

Adding CDF string extractors for a few more election strings.
Leaving out `precinctName` and `districtName` for now, since I need to get clarification on some specifics first.

## Testing Plan
- Unit test coverage

## Checklist

- ~[ ] I have added [logging](https://github.com/votingworks/vxsuite/tree/main/libs/logging) where appropriate to any new user actions, system updates such as file reads or storage writes, or errors introduced.~

- ~[ ] I have added a screenshot and/or video to this PR to demo the change~
- ~[ ] I have added the "user_facing_change" label to this PR to automate an announcement in #machine-product-updates~

### CODE REVIEW

File: libs/utils/src/extract_cdf_ui_strings.test.ts [hunk 1]
```diff
@@ -72,26 +72,198 @@ const tests: Record<ElectionStringKey, () => void> = {
...
+      Election: [
+        {
+          ...ORIGINAL_ELECTION,
+          Candidate: [
+            {
+              ...assertDefined(originalCandidates[0]),
+              '@id': 'candidate1',
+              BallotName: buildInternationalizedText({
+                [LanguageCode.ENGLISH]: 'Pinky and The Brain',
+                [LanguageCode.SPANISH]: 'Pinky y The Brain',
```
Comment (arsalansufi) [inline libs/utils/src/extract_cdf_ui_strings.test.ts:88]: 😆

**Review by arsalansufi — APPROVED** [review]
Excellent examples 😆 - LGTM!

File: libs/utils/src/extract_cdf_ui_strings.test.ts (hunk 1 above)
Comment (eventualbuddha) [reply libs/utils/src/extract_cdf_ui_strings.test.ts:88]: Except it looks like it's ["Pinky y Cerebro"](https://www.youtube.com/watch?v=HKbPt5RcTXk) 🧠

Comment (kofi-q) [reply libs/utils/src/extract_cdf_ui_strings.test.ts:88]: Omg amazing 😆 - must fix immediately

Also, hearing the theme song in Spanish just made my day

**Pull request merged**